tox.ini

```diff
@@ -42,10 +42,12 @@ commands_pre =
         {env:INTEGRATION_TESTS_REPO:git@github.com:gridsingularity/gsy-backend-integration-tests.git} \
         {envtmpdir}/gsy-backend-integration-tests
     ln -sf {envtmpdir}/gsy-backend-integration-tests/gsy_e_sdk_tests/integration_tests/ ./integration_tests
+    ln -sf {envtmpdir}/gsy-backend-integration-tests/docker_compose/ ./docker_compose
```

Comment:
Yeah, also here it shows that [my comment](https://github.com/gridsingularity/gsy-backend-integration-tests/pull/44/files#r869942093) is valid IMO. It could be read as a copy command of some docker-compose library. Do you agree?

Reply:
I agree :)